springboot/src/main/java/ca/jrvs/apps/trading/controller/QuoteController.java

```diff
@@ -0,0 +1,127 @@
+package ca.jrvs.apps.trading.controller;
+
+import ca.jrvs.apps.trading.model.IexQuote;
+import ca.jrvs.apps.trading.model.Quote;
+import ca.jrvs.apps.trading.service.QuoteService;
+import ca.jrvs.apps.trading.util.ResponseExceptionUtil;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+//@Api(value = "quote", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+@Controller
+@RequestMapping("/quote")
+public class QuoteController {
+
+  private QuoteService quoteService;
+
+  @Autowired
+  public QuoteController(QuoteService quoteService) {
+    this.quoteService = quoteService;
+  }
+
+  /**
+   * Update market data
+   */
+  @PutMapping(path= "/iexMarketData")
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public List<Quote> updateMarketData() {
+    try {
+      return quoteService.updateMarketData();
+    } catch (Exception e) {
+      throw ResponseExceptionUtil.getResponseStatusException(e);
+    }
+  }
+
+  /**
+   * Update a given quote in the quote table
+   *  - Manually update a quote in the quote table using IEX market data
+   */
+  @PutMapping(path= "/")
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public Quote putQuote(@RequestBody Quote quote) {
+    try {
+      return quoteService.saveQuote(quote);
+    } catch (Exception e) {
+      throw ResponseExceptionUtil.getResponseStatusException(e);
+    }
+  }
+
+  /**
+   * Add a new ticker to the dailyList (quote table)
+   * - Add a new ticker/symbol to the quote table, so trader can trade this security
+   */
```

Comment:
again, you must use javadoc above a method. please go through your code and fix all